tiledb/sm/rest/rest_client.h

```diff
@@ -270,6 +276,16 @@ class RestClient {
    */
   Status update_attribute_buffer_sizes(
       const serialization::CopyState& copy_state, Query* query) const;
+
+  /**
+   * Helper function encapsulating the functionality of looking up for cached
+   * redirected rest server addresses to avoid the redirection overhead
```

Comment:
Add a new-line before the `@param` lines, similar to the other function comments in this file.